src/Moq/Protected/ProtectedAsMock.cs

```diff
@@ -239,11 +246,21 @@ protected override Expression VisitMethodCall(MethodCallExpression node)
 				{
 					var targetParameter = Expression.Parameter(this.targetType, left.Name);
 					return Expression.Call(targetParameter, FindCorrespondingMethod(node.Method), node.Arguments);
-				}
+				} 
 				else
 				{
-					return node;
+					return base.VisitMethodCall(node);
```

Comment:
†) Is this part of the bugfix mentioned in the changelog? If so, please re-submit as part of a separate bugfix PR.